src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea, ToDraw {
+    private int radius;
+
+    public Circle(int radius) {
```

Comment:
you should pass color to every figure constructor
<img width="320" alt="Screenshot 2022-04-27 at 22 03 50" src="https://user-images.githubusercontent.com/60258373/165600785-574ddea5-95a8-42bc-a8f1-4561dd69eb5c.png">